mne/report.py

```diff
@@ -1414,6 +1456,33 @@ def _render_cov(self, cov_fname, info_fname):
                                          show=show)
         return html
 
+    def _render_whitened_evoked(self, evoked_fname, noise_cov):
+        """Show whitened evoked.
+        """
+        global_id = self._get_id()
+
+        evokeds = read_evokeds(evoked_fname, verbose=False)
+
+        html = []
+        for ev in evokeds:
+            noise_cov = regularize(noise_cov, ev.info)
+            ev_white = whiten_evoked(ev, noise_cov, diag=True)
+            global_id = self._get_id()
+
```

Comment:
more surprises @dengemann : 

```py
from mne import read_evokeds, Covariance
from mne.datasets import sample

path = sample.data_path()
cov_fname = path + '/MEG/sample/sample_audvis-cov.fif'

noise_cov = Covariance(cov_fname)

evoked = read_evokeds(path + '/MEG/sample/sample_audvis-ave.fif')
evoked[0].plot_white(noise_cov)
```

gives

![gfp_issue](https://cloud.githubusercontent.com/assets/3817535/7264722/3e9fd354-e88f-11e4-8d03-b4af3d1b47c7.png)

if you know anything off the top of your head let me know. Otherwise, I'll try to figure it out tomorrow.